source/fab/__init__.py

```diff
@@ -0,0 +1,5 @@
+'''
+Flexible build system for scientific software.
+'''
+
+__version__ = '0.1.dev0'
```

Comment:
Just a passing thought - I think we will eventually want to go with a YYYY.MM.D style of versioning (though I doubt it matters in the build up to the first release version)

Reply:
Not that I'm against changing it right now, but why do you think that?

Reply:
From past experience I find having the major-minor style of versioning means you are forever questioning "should this be a major release?" whereas with the date based one it feels more like it gives every release equal(ly insignificant) weighting and takes that decision out of your hands

Reply:
I tend to think the question of whether it is major (i.e. breaks backward compatibility) or not is an important one to ask on every release. However not strongly enough to argue the point. I have changed it and made a guess at when our first release will be.

Reply:
Agreed